app/controllers/concerns/vet360/writeable.rb

```diff
@@ -8,9 +8,9 @@ module Vet360
   module Writeable
     extend ActiveSupport::Concern
 
-    PROFILE_AUDIT_LOG_TYPES = { email: 'update_email_address',
-                                address: 'update_mailing_address',
-                                telephone: 'update_phone_number' }.freeze
+    PROFILE_AUDIT_LOG_TYPES = { email: :update_email_address,
+                                address: :update_mailing_address,
+                                telephone: :update_phone_number }.freeze
```

Comment:
We _could_ add `.with_indifferent_access` to the hash then we wont have to worry about converting to stings or symbols